### PR TITLE
Add job out-of-memory workaround fix to known issues

### PR DESCRIPTION
**Pull request by hdrei**

### CODE REVIEW

File: docs/helpdesk/issues.md [hunk 1]
```diff
@@ -21,3 +21,9 @@ nodes for instance), add `export PMI_NO_PREINITIALIZE=y` to your batch script.
 
 To avoid job startup failures with `[unset]:_pmi_set_af_in_use:PMI ERROR`, add 
 `export PMI_SHARED_SECRET=""` line to your batch script.
+
+## Job out-of-memory issues in `standard` partition
+
+Some nodes of standard partition have been losing memory over time. A fix to detect these nodes (to restart/clean them) is on its way, but meanwhile one can use a workaround to specify the memory required per node close to the theoretical max value of memory of the nodes. Use e.g. `--mem=225G` in your slurm script.
```
Comment (hdrei) [inline docs/helpdesk/issues.md]:
```suggestion
Some nodes of standard partition are losing memory over time. A fix to detect these nodes (to restart/clean them) is on its way, but meanwhile one can use a workaround to specify the memory required per node close to the theoretical max value of memory of the nodes. Use e.g. `--mem=225G` in your slurm script.
```

Comment (hdrei) [reply docs/helpdesk/issues.md]: Added a suggestion to change the wording from past tense to present

**Review by mszpindler — COMMENTED** [review]
I would rather say nodes have been leaking memory.

File: docs/helpdesk/issues.md [hunk 2]
```diff
@@ -21,3 +21,9 @@ nodes for instance), add `export PMI_NO_PREINITIALIZE=y` to your batch script.
 
 To avoid job startup failures with `[unset]:_pmi_set_af_in_use:PMI ERROR`, add 
 `export PMI_SHARED_SECRET=""` line to your batch script.
+
+## Job out-of-memory issues in `standard` partition
+
+Some nodes of standard partition are losing memory over time. A fix to detect these nodes (to restart/clean them) is on its way, but meanwhile one can use a workaround to specify the memory required per node to something that should be available. Use e.g. `--mem=225G` in your slurm script.
```
Comment (hdrei) [inline docs/helpdesk/issues.md]:
```suggestion
Some nodes of standard partition are leaking memory over time. A fix to detect these nodes (to restart/clean them) is on its way, but meanwhile one can use a workaround to specify the memory required per node to something that should be available. Use e.g. `--mem=225G` in your slurm script.
```

**Review by mszpindler — APPROVED** [review]

**Review by emmanuelumi — APPROVED** [review]

**Pull request merged**